### PR TITLE
Golem

### PR DESCRIPTION
**Pull request by gillespied**
Moved the code base into golem framework.

### CODE REVIEW

**Review by DIPAD-Sara-L-Smith — APPROVED** [review]

File: R/fct_plot.R [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# This file probably needs a name change.
```
Comment (CSH-Sara-Smith) [inline R/fct_plot.R]: Agreed re: name change; maybe something like fct_xts, fct_prepare_xts, or fct_cleandata?

File: example-scripts/readme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Example Scripts 
+
+In what I think is an increasingly bad decision we decided to allow people to use R scripts to upload data. This folder contains a few simple values
+
+We need to think about what packages are available to these scripts and how to limit them from breaking everything. Can they be run in a way which will not effect the enviroment.
```
Comment (CSH-Sara-Smith) [inline example-scripts/readme.md:5]: You're right: we don't want the users to affect the environment. There isn't really any reason to build this functionality into the tool, actually - any data cleaning scripts can just be run separately beforehand and the outputs saved as csvs (or equivalent). The data only needs to be cleaned once, anyway, and with your recent changes users can more easily upload multiple files at once, so there's no need to combine files in a separate script.

**Review by DIPAD-Sara-L-Smith — CHANGES_REQUESTED** [review]
I'm experiencing an error when I try to append one data file to another - the app becomes non-responsive ("greys out")? The console only says this:

`Use `spec()` to retrieve the guessed column specification
Pass a specification to the `col_types` argument to quiet this message
[1] "Loaded a valid dataframe"
[1] "Merged 1 dataframes together."
Warning: Error in : `.x` is empty, and no `.init` supplied
  [No stack trace available]`

File: example-scripts/readme.md (hunk 2 above)
Comment (gillespied) [reply example-scripts/readme.md:5]: This is coming from the code I wrote to try and append more meaningful suffixes when joining dfs together. It looks like I didnt check all the functionality before deciding I'd fixed it. I'll see if I can get it working and if not drop back to the commit which just added x or y etc.

Comment (gillespied) [reply example-scripts/readme.md:5]: I reverted to the old one for now. If we think we need better names we can discuss. Its trickier than I thought to keep track of where the columns came from.

**Comment by gillespied** [comment]
I've made the changes you requested.

**Review by DIPAD-Sara-L-Smith — APPROVED** [review]

**Pull request merged**